### PR TITLE
basic game terrain

### PR DESCRIPTION
**Pull request by Oreo4102004**
-> added two controllable rectangles as placeholder for future sprites
-> basic movements and jump

### CODE REVIEW

File: src/game.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        if self.x < 0:
+            self.x = 0
+
+
+# creating and pushing objects to the screen
+
+
+player = Player(
+    x=0, y=HEIGHT // 6, ground=HEIGHT // 6, width=40, height=40, color=(235, 64, 52)
```
Comment (brad90four) [inline src/game.py:53]:
```suggestion
    x=0, y=HEIGHT // 6, ground=HEIGHT // 6, width=40, height=40, color=(240, 100, 50)
```
Maybe player 1 can be a dark blue?

File: src/game.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    def update(self, dt):
+        # movements
+        if self.key_handler[pyglet.window.key.A]:
+            self.x -= 250 * dt
+
+        if self.key_handler[pyglet.window.key.D]:
+            self.x += 250 * dt
+
+        # jump logic
+        if self.key_handler[pyglet.window.key.W]:
```
Comment (brad90four) [inline src/game.py:26]: Should the jump be "W" or "SPACE"?

**Review by brad90four — APPROVED** [review]
LGTM. Just a suggestion on the player 1 color and if the jump should be W or Space bar

File: src/game.py (hunk 2 above)
Comment (Oreo4102004) [reply src/game.py:26]: space seems the accurate key for jump , but W is easier to control with so i will keep it W for now 
also the jump logic may have some bugs to feel free to fix 👍

**Review by AbooMinister25 — APPROVED** [review]
LGTM

**Pull request merged**